setup.py

```diff
@@ -21,7 +21,8 @@
 with open("README.md") as f:
     long_description = f.read()
 with open('requirements/requirements.txt') as fp:
-    install_requires = fp.read()
+    install_requires = fp.read().splitlines()
+    install_requires.append("futures; python_version == '2.7'")
```

Comment:
Shouldn't this be part of the requirements.txt instead of being hard coded here?

Reply:
Yes, a good catch.